configure.ac

```diff
@@ -671,9 +666,9 @@ if test "$enable_cloud" != "no" -a "$aclk_ng" != "yes"; then
 
     if test "${with_bundled_lws}" = "yes"; then
         AC_MSG_CHECKING([if libwebsockets static lib is present])
-        if test -f "${bundled_lws_dir}/libwebsockets.a"; then
-            LWS_CFLAGS="-I ${bundled_lws_dir}/include"
-            OPTIONAL_LWS_LIBS="${bundled_lws_dir}/libwebsockets.a"
+        if test -f "externaldeps/libwebsockets/libwebsockets.a"; then
+            LWS_CFLAGS="-I \$(abs_top_srcdir)/externaldeps/libwebsockets/include"
+            OPTIONAL_LWS_LIBS="externaldeps/libwebsockets/libwebsockets.a"
```

Comment:
why `OPTIONAL_LWS_LIBS` has neigher `abs_top_srcdir` or build dir?

Reply:
I plan on doing this in a follow-up PR. I wanted to get a thumbs up first, to make sure everyone's fine with the approach I'm taking. Considering that making the internal library deps use an absolute path is a small change, I can update this PR if you think it's easier/better to review at once.